### ISSUE TITLE
Integrate ConEmu and/or Commander

### ISSUE THREAD

**Issue by hohwille** [devonfw/devon-ide#62]
With `oasp4j-ide` we could just deliver a zip with the software folder included with e.g. `Commander` included. We can still do that with `devon-ide` but the cool thing is its new lightweight approach to download required or requested software on demand. This also allows to get the right software for your current OS and make the solution portable. Even further this approach allows upgrading to newer versions much easier.
So should we create a commandlet for ConEmu and/or Commander?

**Comment by markusschuh** [comment]
I personally would not do that investment - especially not for such non-portable tools, which are only loosely coupled with devonfw. It might help, if a developer only develops with devonfw - but in case of not, it would make far more sense, to have the tool outside / independent of devon-ide - instead of using two different conemu inside / outside devonfw.

I personally still prefer to install those tools - and keep them up-to-date - via the free package management solutions under windows. That means
Either [chocolatey](www.chocolatey.org) or [scoop](https://scoop.sh)

Of course those package manager have their own constraints - but the same is true for devon-ide.

**Comment by hohwille** [comment]
@markusschuh thanks for your feedback. For #61 I came to the same conclusion.
When someone still creates a custom ZIP distribution based on `devon-ide` he can of course integrate commander in the same way as we did for `oasp4j-ide`.
However, for the default we should rather invest into documentation:
Long time ago I already did this for ConEmu that I also used to install globally on my system:
https://github.com/devonfw/devon-ide/wiki/integration#conemu
